tools/jetpack-live-branches/jetpack-live-branches.user.js

```diff
@@ -147,16 +147,14 @@
 							);
 							return;
 						}
-					} else if ( body.code === 'rest_no_route' ) {
+					} else {
 						plugins.push( {
 							name: 'branch',
 							value: currentBranch,
 							label: 'Jetpack',
 							checked: true,
 							disabled: true,
 						} );
```

Comment:
The case you kept was for when we first added multiple-branch support. Probably best to keep the error case instead.
```suggestion
					} else {
						throw new Error( 'Invalid response from server' );
```

Reply:
Makes sense, I was thinking best to default vs throw an error but this aligns to current functionality.